updating the readme to add clarity to cert docs and developer testing

acornett21 opened the pull request:
- Updating readme to point to official certification docs for clarity
- Updating the testing section so parters know these are not tests they run, but tests for developers.
- [JIRA](https://issues.redhat.com/browse/EET-3216)

acornett21 commented:
@taylorjordanNC Please take a look at let us know if you have more suggestions. Thanks.

taylorjordanNC commented:
lgtm!

skattoju commented on README.md:
```diff
@@ -13,6 +13,9 @@
 This project is in active and rapid development! Many facets of this project are
 subject to change, and some features are not fully implemented.
 
```
May be add a section header ex: `## Certification Documentation` or something like that ?

acornett21 replied:
I added a header with the same name that is used on the website.

skattoju reviewed (changes requested):
A section header might help it stand out ex: ## Certification Workflow Docs or something like that ?

acornett21 commented:
/retest

acornett21 commented:
> This looks like it does more than just update for the official docs? Do we want to split this into two PRs?
@bcrochet Both of these asks were in the JIRA (which I added above). I've updated the title of this PR for clarity, since it only pulled the first half of the commit message.

bcrochet reviewed (approved):
/lgtm
